lib/core/app-config/schema.json

```diff
@@ -1145,6 +1145,25 @@
             }
           }
         },
+        "highlight": {
+          "prefix": "string",
+          "postfix": "string",
+          "snippetCount": 0,
```

Comment:
All these properties should be under the "properties" inside the schema json under a different structure

Reply:
done , thanks 👍